source/reference/failure-recovery.txt

```diff
@@ -1,8 +1,8 @@
 .. _c2c-failure-recovery:
 
-==========================
-Recovery From Sync Failure
-==========================
+=====================================
+Recovery Steps for Disaster Recovery 
```

Comment:
[suggestion] This is really a tough page title. Perhaps, "Use `mongosync` for Disaster Recovery" although the question then becomes, how do you set up `mongosync` for disaster recovery in the first place?

Reply:
Fixed